x-pack/plugins/enterprise_search/public/applications/shared/result/types.ts

```diff
@@ -9,6 +9,7 @@ import { IconType } from '@elastic/eui';
 
 export interface MetaDataProps {
   id: string;
+  title?: string;
   onDocumentDelete?: Function;
```

Comment:
...huh, why is there an onDocumentDelete function in this metadata type? Going through the codebase I can't find it being assigned anywhere. Can we delete this while we're at it? Even if we need to delete a specific document, I don't think passing an onDocumentDelete through metaData is what we should be doing. (cc @efegurkan for opinions, because it looks like he built this)

Reply:
This component is mostly @daveyholler's code I just integrated it in to our codebase. 

The original stub had a button on metadata popover to delete the document.
![Screenshot 2022-08-25 at 14 11 57](https://user-images.githubusercontent.com/1410658/186661071-361f1c82-ac02-40b6-9688-4354e8bd6c6b.png)

I think we delete this as our current documentation doesn't need it. If it is needed we bring it back or find a different solution.

Reply:
I'll remove it for now and we can add it back when we actually need it.

Reply:
I'm approving anyway because this isn't a blocking thing by any means, though :)